bigtable/client/instance_admin.h

```diff
@@ -112,6 +112,18 @@ class InstanceAdmin {
    */
   std::vector<google::bigtable::admin::v2::Cluster> ListClusters();
 
+  /**
+   * Deletes the specified cluster of an instance in the project.
+   * @param instance_id the id of the instance in the project
+   * @param cluster_id the id of the cluster in the project that needs to be
+   *   deleted
+   *
+   *  **Example**
+   *  @snippet bigtable_samples_instance_admin.cc delete cluster
+   */
+  void DeleteCluster(std::string const& instance_id,
```

Comment:
Please use a strong type for `instance_id` and `cluster_id` here, too easy to get the arguments backwards.

Reply:
@coryan ,
currently, strong types are defined in `table_admin_strong_types.h`.  `ClusterId` is already defined here. Should I rename it to `bigtable_strong_types.h` ?
`InstanceId` can also be part of it and name wont be misleading when included in InstanceAdmin

Reply:
That sounds like a good idea.  Thanks!

Reply:
fixed